src/utils.ts

```diff
@@ -10,6 +10,48 @@ export interface Options {
   etherscan: string | undefined;
 }
 
+export function getLink(chain: ChainName, hash: string): string {
+  if (!hash) {
+    return "";
+  }
+  if (chain.includes("ethereum")) {
+    return `https://etherscan.io/tx/${hash}`;
+  } else if (chain.includes("polygon")) {
+    return `https://polygonscan.com/tx/${hash}`;
+  } else if (chain.includes("optimism")) {
+    return `https://optimistic.etherscan.io/tx/${hash}`;
+  } else if (chain.includes("arbitrum")) {
+    return `https://arbiscan.io/tx/${hash}`;
+  }
```

Comment:
Since the testnet txns won't show on the mainnet explorers, we'll need to handle those as well. 
- goerli: https://goerli.etherscan.io/tx/
- mumbai: https://mumbai.polygonscan.com/tx/
- optimism kovan: https://kovan-optimistic.etherscan.io/tx/
- optimism goerli: https://blockscout.com/optimism/goerli/tx/
- arbitrum goerli: https://goerli-rollup-explorer.arbitrum.io/tx/

the `includes` will still be good, since it will catch the staging chains too